htdocs/core/js/lib_head.js.php

```diff
@@ -109,11 +109,11 @@
 // Javascript libraries for Dolibarr ERP CRM (https://www.dolibarr.org)
 
 // For jQuery date picker
-var tradMonths = <?php echo json_encode($tradMonths) ?>;
-var tradMonthsShort = <?php echo json_encode($tradMonthsShort) ?>;
-var tradDays = <?php echo json_encode($tradDays) ?>;
-var tradDaysShort = <?php echo json_encode($tradDaysShort) ?>;
-var tradDaysMin = <?php echo json_encode($tradDaysMin) ?>;
+var tradMonths = '<?php echo json_encode($tradMonths) ?>';
```

Comment:
json_encode return an array string that is a javascript array too
["Janvier","Fevrier","Mars","Avril","Mai","Juin","Juillet","Aout","Septembre","Octobre","Novembre","Decembre"]
If you add quotes, tradMonths will become a string and not an array.